node/src/service.rs

```diff
@@ -353,10 +356,6 @@ where
 		bool,
 	) -> Result<Box<dyn ParachainConsensus<Block>>, sc_service::Error>,
 {
-	if matches!(parachain_config.role, Role::Light) {
```

Comment:
dont we need that anymore?

Reply:
Role::Light was removed from 28 .. therefore we do not need this anymore .. @mateuszaaa